File: Python/Q60.py
# Q60.py - Program to Store Information of a Student Using Structure
class Student:
    def __init__(self, name, age, roll):
        self.name = name
        self.age = age
        self.roll = roll

name = input("Enter name: ")
age = int(input("Enter age: "))
roll = int(input("Enter roll number: "))
student = Student(name, age, roll)
print(f"Student info - Name: {student.name}, Age: {student.age}, Roll: {student.roll}")

File: C/Q60.c
#include <stdio.h>
struct Student {
    int rollNo;
    char name[100];
    float CGPA;
    char branch[50];

};

int main() {
    struct Student s;
    printf("Enter roll number: ");
    scanf("%d", &s.rollNo);

    printf("Enter name: ");
    scanf("%s", s.name);

    printf("Enter branch: ");
    scanf("%s", s.branch);

    printf("Enter CGPA: ");
    scanf("%f", &s.CGPA);

    printf("Student Info: \n");
    printf("Roll Number: %d\n", s.rollNo);
    printf("Name: %s\n", s.name);
    printf("CGPA: %.2f\n", s.CGPA);
    printf("Branch: %s\n", s.branch);

    return 0;
}